modelplugin/aether-2.0.0/yang/security-profile@2020-11-30.yang

```diff
@@ -0,0 +1,72 @@
+module security-profile {
+  namespace "http://opennetworking.org/aether/security-profile";
+  prefix pro;
+
+  organization "Open Networking Foundation.";
+  contact "Scott Baker";
+  description "To generate JSON from this use command
+    pyang -f jtoxx test1.yang | python3 -m json.tool > test1.json
+    Copied from YangUIComponents project";
+
+  revision "2020-11-30" {
+    description "An Aether Security Profile";
+    reference "RFC 6087";
+  }
+
+  // TODO: Think more on whether this should be a UUID or
+  // simply a unique name. If it's a UUID, could fix the 
+  // string length.
+  typedef security-profile-id {
+        type string {
+            length 1..32;
+        }
+  }
+
+  container security-profile {
+    description "The top level container";
+
+    list security-profile {
+      key "id";
+      description
+        "List of security profiles";
+
+      leaf id {
+        type security-profile-id;
+        description "ID for this security profile.";
+      }
+
+      leaf display-name {
+        type string {
+            length 1..80;
+        }
+        description "display name to use in GUI or CLI";
+      }     
+
+      leaf key {
+        type string {
+          length 1..32;
+        }
+        description "Secret key for USIM";
+      }
+
+      leaf opc {
```

Comment:
these leaf names are quite terse - are they coming from some standard? `opc` does not mean much to me